programs/CMakeLists.txt

```diff
@@ -112,6 +112,8 @@ add_subdirectory (obfuscator)
 add_subdirectory (install)
 add_subdirectory (git-import)
 
+#add_subdirectory (grpc-client)
```

Comment:
Can I build it?

Reply:
It's an old unfinished C++ client, it cannot be build. So we can finish it or remove it.

Reply:
I removed this target in https://github.com/ClickHouse/ClickHouse/pull/17435